tests/Integration/TestCase.php

```diff
@@ -187,10 +271,232 @@ public function go_to_new_post( string $post_status = 'publish' ): int {
+			throw new RiskyTestError( 'Function assert_script_statuses() has been used without any arguments' );
+		}
+
+		foreach ( $assert_true as $status ) {
+			self::assertTrue(
+				wp_script_is( $handle, $status ),
+				"Unexpected script status: $handle status should be '$status'"
+			);
+		}
+
+		foreach ( $assert_false as $status ) {
+			self::assertFalse(
+				wp_script_is( $handle, $status ),
+				"Unexpected script status: $handle status should NOT be '$status'"
+			);
+		}
+	}
+
+	/**
+	 * Asserts that a passed style is not registered.
+	 *
+	 * @param string $handle Style handle to test.
+	 */
+	public function assert_is_style_not_registered( string $handle ): void {
+		$this->assert_style_statuses( $handle, array(), array( 'registered' ) );
+	}
+
+	/**
+	 * Asserts that a passed style is registered.
+	 *
+	 * @param string $handle Style handle to test.
+	 */
+	public function assert_is_style_registered( string $handle ): void {
+		$this->assert_style_statuses( $handle, array( 'registered' ) );
+	}
+
+	/**
+	 * Asserts that a passed style is not enqueued.
+	 *
+	 * @param string $handle Style handle to test.
+	 */
+	public function assert_is_style_not_enqueued( string $handle ): void {
+		$this->assert_style_statuses( $handle, array(), array( 'enqueued' ) );
+	}
+
+	/**
+	 * Asserts that a passed style is enqueued.
+	 *
+	 * @param string $handle Style handle to test.
+	 */
+	public function assert_is_style_enqueued( string $handle ): void {
+		$this->assert_style_statuses( $handle, array( 'enqueued' ) );
+	}
+
+	/**
+	 * Asserts multiple enqueuing statuses for a style.
+	 *
+	 * @param string $handle       Style handle to test.
+	 * @param array  $assert_true  Optional. Statuses that should assert to true. Accepts 'enqueued',
+	 *                             'registered', 'queue', 'to_do', and 'done'. Default is an empty array.
+	 * @param array  $assert_false Optional. Statuses that should assert to false. Accepts 'enqueued',
+	 *                             'registered', 'queue', 'to_do', and 'done'. Default is an empty array.
+	 *
+	 * @throws RiskyTestError If no assertions ($assert_true, $assert_false) get passed to the function.
+	 */
+	private function assert_style_statuses( string $handle, array $assert_true = array(), array $assert_false = array() ): void {
+		if ( 0 === count( $assert_true ) + count( $assert_false ) ) {
+			throw new RiskyTestError( 'Function assert_style_statuses() has been used without any arguments' );
+		}
+
+		foreach ( $assert_true as $status ) {
+			self::assertTrue(
+				wp_style_is( $handle, $status ),
+				"Unexpected style status: $handle status should be '$status'"
+			);
+		}
+
+		foreach ( $assert_false as $status ) {
+			self::assertFalse(
+				wp_style_is( $handle, $status ),
+				"Unexpected style status: $handle status should NOT be '$status'"
+			);
+		}
+	}
+
+	/**
+	 * Return TRUE if minimum PHP version is 7.2 or higher. We uses this if something works
```

Comment:
We still have function summaries that need to be converted to third person. Please check other places as well, as I there seem to be occurrences in multiple places.